README.md

```diff
@@ -1,25 +1,47 @@
-# Responsive Media
+# Intro to Responsive Media
 
-## Overview
+## Problem Statement
 
-In this lesson we will look at a strategy for making our media content responsive.
+As we've learned with media queries, people will be viewing your site from a
+number of different types of devices. Some media we may want to stretch or shrink
```

Comment:
Is "We may want some media to stretch or shrink..." easier to read or is it just me?